Content.Server/Intellicard/IntellicardSystem.cs

```diff
@@ -0,0 +1,64 @@
+using Content.Server.Ghost.Roles;
+using Content.Server.RandomMetadata;
+using Content.Shared.Intellicard;
+using Content.Shared.Popups;
+using Content.Shared.Silicons.StationAi;
+using Content.Shared.Verbs;
+
+namespace Content.Server.Intellicard;
+
+/// <summary>
+/// System for handling the behaviour of intellicards.
+/// </summary>
+public sealed class IntellicardSystem : SharedIntellicardSystem
+{
+    [Dependency] private readonly   SharedPopupSystem _popup = default!;
+    [Dependency] private readonly   ToggleableGhostRoleSystem  _ghostRole = default!;
+    [Dependency] private readonly   SharedStationAiSystem _stationAi = default!;
+    [Dependency] private readonly   MetaDataSystem _metaData = default!;
+    [Dependency] private readonly   RandomMetadataSystem  _randomMeta = default!;
```

Comment:
There probably shouldn't be a 3-space padding between the types.

```suggestion
    [Dependency] private readonly SharedPopupSystem _popup = default!;
    [Dependency] private readonly ToggleableGhostRoleSystem  _ghostRole = default!;
    [Dependency] private readonly SharedStationAiSystem _stationAi = default!;
    [Dependency] private readonly MetaDataSystem _metaData = default!;
    [Dependency] private readonly RandomMetadataSystem  _randomMeta = default!;
```

Reply:
That's because in that file, there is a line with `protected readonly`. This slightly changed the alignment, so they added two extra spaces to the rest so that all the types were aligned.

https://github.com/space-wizards/space-station-14/blob/647db6aa878ce3ae6c10ec2bbf5b8b99665935ff/Content.Shared/Silicons/StationAi/SharedStationAiSystem.cs#L48-L50

Though as you don't have that, it's probably best to conform to the regular style and separate them all with only 1 space.